test/int/integration_suite_test.go

```diff
@@ -71,27 +72,23 @@ const (
 var (
 	// This variable is "global" - as is visible only on the first ginkgo node
 	testEnv *envtest.Environment
+	cfg     *rest.Config
```

Comment:
What is this config about? Is this config immutable and the same for all tests, even concurrent ones?